apps/client/src/features/rundown/time-input-flow/TimeInputFlow.tsx

```diff
@@ -88,7 +88,7 @@ const TimeInputFlow = (props: EventBlockTimerProps) => {
         disabled={isLockedDuration}
         placeholder='End'
       >
-        <Tooltip label='Lock end' openDelay={tooltipDelayMid}>
+        <Tooltip label='Lock end' openDelay={tooltipDelayMid} as='button'>
```

Comment:
question: should the as be in the `InputRightElement`?

Reply:
Resolving with https://github.com/cpvalente/ontime/pull/1228/commits/3b948e6b29a2dffc36a6e0ce2a878bdbcd378272